### ISSUE TITLE
fatal error: 'Flutter/Flutter.h' file not found

### ISSUE THREAD

**Issue by z85510** [flutter/flutter#79708]
I got this error and I tried these : 

```
/development/flutter/.pub-cache/hosted/pub.dartlang.org/webview_flutter-2.0.3/ios/Classes/JavaScriptChannelHandler.h:5:9: fatal error: 'Flutter/Flutter.h' file not found
#import <Flutter/Flutter.h>

```
- flutter clean 
- delete ios folder
- flutter create .
- pub cache repair

I don't have any problem with android but in iOS I can't build and run anymore.

<details>
  <summary> dependencies </summary>

```
dependencies:
  flutter:
    sdk: flutter

  flutter_localizations:
    sdk: flutter
  fluttercontactpicker: ^4.4.0
  flutter_sms_inbox: ^0.1.1+1
  flutter_svg: ^0.21.0-nullsafety.0
  loading: ^1.0.2
  flutter_share: ^1.0.5
  permission_handler: ^6.1.1
  contacts_service: ^0.5.0-nullsafety.0
  quick_actions: ^0.6.0
  introduction_screen: ^2.1.0
  pin_code_fields: ^7.0.0
  bot_toast: ^4.0.1
  percent_indicator: ^3.0.1
  flutter_launcher_icons: ^0.9.0
  country_code_picker: ^2.0.1
  connectivity: ^3.0.3
  date_time_picker: ^2.0.0
  keyboard_actions: ^3.4.0
  fl_chart: ^0.35.0
  super_tooltip: ^1.0.1
  flutter_inapp_purchase: ^5.0.0
  package_info: ^2.0.0
  flutter_secure_storage: ^4.1.0
  after_layout: ^1.1.0
  math_expressions: ^2.1.0
  date_util: ^0.1.4
  cupertino_icons: ^1.0.2
  http: ^0.13.1
  provider: ^4.0.1
  rxdart: ^0.26.0
  shamsi_date: ^0.8.1
  logger: ^1.0.0
  intl: ^0.17.0
  moor: ^4.2.1
  sqlite3_flutter_libs: ^0.4.1
  json_annotation: ^4.0.1
  path: ^1.8.0
  firebase_core: ^1.0.2
  firebase_messaging: ^9.1.0
  firebase_crashlytics: ^1.0.0
  flutter_switch: ^0.3.0
  get_it: ^6.0.0
  url_launcher: ^6.0.2
  provider_architecture: ^1.1.1+1
  focused_menu: ^1.0.5
  device_info: ^2.0.0
  sms_autofill: ^1.3.1
  lottie: ^1.0.1
  carousel_slider: ^3.0.0
  path_provider: ^2.0.1
  flutter_local_notifications: ^5.0.0-nullsafety.0
  in_app_review: ^2.0.1
  link_text: ^0.2.0
  rate_my_app: ^1.1.0+1
  showcaseview: ^0.1.6
  flutter_keyboard_visibility: ^5.0.0
  persian_datepicker: ^1.3.1
  google_sign_in: ^5.0.1
  flutter_html: ^2.0.0-nullsafety.0
  image_picker: ^0.7.4
  photo_view: ^0.11.1
  image: ^3.0.2
  flutter_image_compress: ^1.0.0-nullsafety
  image_cropper: ^1.4.0
  charts_flutter: ^0.10.0
  google_mobile_ads: ^0.12.1
```

</details>

<details>
  <summary>Logs</summary>

Xcode build done.                                           15.2s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **

Xcode's output:
↳
    /Users/aref/development/flutter/.pub-cache/hosted/pub.dartlang.org/wakelock-0.4.0/ios/Classes/messages.m:4:9: fatal error: 'Flutter/Flutter.h' file not found
    #import <Flutter/Flutter.h>
            ^~~~~~~~~~~~~~~~~~~
    1 error generated.
    In file included from /Users/aref/development/flutter/.pub-cache/hosted/pub.dartlang.org/webview_flutter-2.0.3/ios/Classes/JavaScriptChannelHandler.m:5:
    /Users/aref/development/flutter/.pub-cache/hosted/pub.dartlang.org/webview_flutter-2.0.3/ios/Classes/JavaScriptChannelHandler.h:5:9: fatal error: 'Flutter/Flutter.h' file not found
    #import <Flutter/Flutter.h>
            ^~~~~~~~~~~~~~~~~~~
    1 error generated.
    In file included from /Users/aref/development/flutter/.pub-cache/hosted/pub.dartlang.org/webview_flutter-2.0.3/ios/Classes/FLTWebViewFlutterPlugin.m:5:
    /Users/aref/development/flutter/.pub-cache/hosted/pub.dartlang.org/webview_flutter-2.0.3/ios/Classes/FLTWebViewFlutterPlugin.h:5:9: fatal error: 'Flutter/Flutter.h' file not found
    #import <Flutter/Flutter.h>
            ^~~~~~~~~~~~~~~~~~~
    1 error generated.
    In file included from /Users/aref/development/flutter/.pub-cache/hosted/pub.dartlang.org/webview_flutter-2.0.3/ios/Classes/FLTWKProgressionDelegate.m:5:
    /Users/aref/development/flutter/.pub-cache/hosted/pub.dartlang.org/webview_flutter-2.0.3/ios/Classes/FLTWKProgressionDelegate.h:5:9: fatal error: 'Flutter/Flutter.h' file not found
    #import <Flutter/Flutter.h>
            ^~~~~~~~~~~~~~~~~~~
    1 error generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

Could not build the application for the simulator.
Error launching application on iPhone 12 Pro Max.

</details>

**Comment by habibalkhabbaz** [comment]
+1

**Comment by z85510** [comment]
> +1
> The problem occurs on version 2.0.4. Downgrading to 2.0.3 solves the problem

I didn't use that and it's not in pubspec.yaml 
but in pubspec.lock I found this : 
```
 webview_flutter:
    dependency: transitive
    description:
      name: webview_flutter
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.0.3"
```
How can I find which dependency use it and how can I solve this problem ?

**Comment by nt4f04uNd** [comment]
duplicate of https://github.com/flutter/flutter/issues/79575

**Comment by iapicca** [comment]
possible duplicate of https://github.com/flutter/flutter/issues/79404

**Comment by z85510** [comment]
> possible duplicate of #79404

for me it doesn't work, runOnlyForDeploymentPostprocessing is 0 and I removed crashlytics as well but it's not working.

**Comment by z85510** [comment]
something weird is that sometimes the error isn't for webview_flutter and it's for another library : 
`
.pub-cache/hosted/pub.dartlang.org/flutter_sms_inbox-0.1.1+1/ios/Classes/FlutterSmsInboxPlugin.h:1:9: fatal error: 'Flutter/Flutter.h' file not found`

or 
`
.pub-cache/hosted/pub.dartlang.org/wakelock-0.4.0/ios/Classes/WakelockPlugin.h:1:9: fatal error: 'Flutter/Flutter.h' file not found`

**Comment by z85510** [comment]
I have found out the solution, it was because of my location ! 
I'm in Iran and we can't use google products without using VPN but from last days we couldn't use it even using VPN, I found another VPN and I figured out that some files couldn't download completely , also there wasn't any error while downloading
